src/renderer/util/guessClipBoardFilePath.js

```diff
@@ -0,0 +1,26 @@
+import { isInElectron, isOsx, isWin } from '../config'
+import plist from 'plist'
+import electron from 'electron'
+
+export const guessClipboardFilePath = () => {
```

Comment:
Please refactor this fcuntions. E.g. remove inner functions and `electron.remote` to `{ ..., remote } import 'electron'`.